src/message.rs

```diff
@@ -1783,22 +1816,30 @@ pub async fn delete_msgs_ex(
     }
     res?;
 
+    if !deleted_info_msgs.is_empty() {
+        if modified_chat_ids.len() > 1 {
+            bail!("Can only delete from same chat.");
+        }
+        let (info_msg_ids, info_msg_rfc724): (Vec<_>, Vec<_>) =
+            deleted_info_msgs.into_iter().unzip();
+
+        for &msg_id in &info_msg_ids {
+            let msg = Message::load_from_db(context, msg_id).await?;
+            delete_msg_locally(context, &msg).await?;
+        }
+
+        if let Some(chat_id) = modified_chat_ids.iter().next() {
+            send_delete_request(context, chat_id, &info_msg_rfc724).await?;
+        }
+    }
+
     if delete_for_all {
         ensure!(
             modified_chat_ids.len() == 1,
             "Can delete only from same chat."
         );
         if let Some(chat_id) = modified_chat_ids.iter().next() {
-            let mut msg = Message::new_text("🚮".to_owned());
-            // We don't want to send deletion requests in chats w/o encryption:
-            // - These are usually chats with non-DC clients who won't respect deletion requests
-            //   anyway and display a weird trash bin message instead.
-            // - Deletion of world-visible unencrypted messages seems not very useful.
```

Comment:
Did you mean to remove this comment because it's wrong/unnecessary, or accidentally?